test/integrations/destinations/salesforce_oauth/dataDelivery/oauth.ts

```diff
@@ -0,0 +1,105 @@
+import { ProxyMetdata } from '../../../../../src/types';
+import { ProxyV1TestData } from '../../../testTypes';
+import { generateProxyV1Payload } from '../../../testUtils';
+
+const commonHeadersForWrongToken = {
+  Authorization: 'Bearer expiredAccessToken',
+  'Content-Type': 'application/json',
+};
+const params = { destination: 'salesforce_oauth' };
+
+const users = [
+  {
+    Email: 'danis.archurav@sbermarket.ru',
+    Company: 'itus.ru',
+    LastName: 'Danis',
+    FirstName: 'Archurav',
+    LeadSource: 'App Signup',
+    account_type__c: 'free_trial',
+  },
+];
+
+const statTags = {
+  retryable: {
+    destType: 'SALESFORCE_OAUTH',
+    destinationId: 'dummyDestinationId',
+    errorCategory: 'network',
+    errorType: 'retryable',
+    feature: 'dataDelivery',
+    implementation: 'native',
+    module: 'destination',
+    workspaceId: 'dummyWorkspaceId',
+  },
+};
+
+const commonRequestParameters = {
+  headers: commonHeadersForWrongToken,
+  JSON: users[0],
+  params,
+};
+
+export const proxyMetdataWithSecret: ProxyMetdata = {
+  jobId: 1,
+  attemptNum: 1,
+  userId: 'dummyUserId',
+  sourceId: 'dummySourceId',
+  destinationId: 'dummyDestinationId',
+  workspaceId: 'dummyWorkspaceId',
+  secret: {
+    access_token: 'expiredAccessToken',
+    instanceUrl: 'https://rudderstack.my.salesforce_oauth.com',
+  },
+  destInfo: { authKey: 'dummyDestinationId' },
+  dontBatch: false,
+};
+
+export const reqMetadataArrayWithSecret = [proxyMetdataWithSecret];
+
+export const testScenariosForV1API: ProxyV1TestData[] = [
+  {
+    id: 'salesforce_v1_scenario_1',
+    name: 'salesforce_oauth',
+    description: '[Proxy v1 API] :: Test with expired access token scenario',
```

Comment:
Add a positive and invalid access case

Reply:
expired or invalid session id response is same